test/java/seedu/addressbook/commands/AddCommandTest.java

```diff
@@ -95,6 +95,7 @@ private void assertConstructingInvalidAddCmdThrowsException(String name, String
         fail(error);
     }
 
+
```

Comment:
Stray line break? It is a good practice to check the diff of the PR and fix any problems before submitting it.